src/test/java/core/basesyntax/impl/ReportCreatorTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportCreator;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReportCreatorTest {
+    private static final String EXPECTED_REPORT = "fruit,quantity\nbanana,15\napple,25";
```

Comment:
please make this local variable or add  String expectedReport = "fruit,quantity"; on class level too